### DISCUSSION TITLE
Does KEDA 2.14 not support K8s 1.30?

### DISCUSSION THREAD

**Discussion by qpetraroia** [kedacore/keda#5775] [Q&A / Need Help] [answered]
I thought KEDA 2.14 was going to support K8s 1.30?

**Comment by tomkerkhove** [comment, marked as answer]
As per https://keda.sh/docs/2.14/operate/cluster/#kubernetes-compatibility, no.

The reason is that Kubernetes v1.30 was released a week before our release and we could not validate the support for in that timeframe nor is there a Kind image for it up until today.
